Task_DEV_4/Task_DEV_4/Parallelogram.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+namespace Task_DEV_4
+{
+    class Parallelogram : Square
+    {
+        public double HeldHeight { get; set; }
+        public double Angle { get; set; }
+        public double Side { get; set; }
+
+        public Parallelogram(double length, double heldHeight) : base(length)
+        {
+            HeldHeight = heldHeight;
+        }
+
+        public Parallelogram(double length, double angle, double side) : base(length)
+        {
+            Length = length;
+            Side = side;
+            Angle = angle;
+        }
+
+        public override  double CalculateArea()
+        {
+            if(HeldHeight > 0)
```

Comment:
Why do you check only HeldHeight. Length can be 0 too.

Reply:
Done.